fix(editor): strip leading slash for bin path on windows

Sysix opened the pull request:
The `bin` variable is under windows `\C:\Users\..`, using this path style to check accessibility,
will check a file like: `C:\C:\Users\...` which is of course invalid.

camc314 commented:
### Merge activity

* **Sep 15, 1:22 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

Sysix committed:
fix(editor): strip leading slash for bin path on windows (#13738)

The `bin` variable is under windows `\C:\Users\..`, using this path style to check accessibility,
will check a file like: `C:\C:\Users\...` which is of course invalid.
